index.md

```diff
@@ -236,19 +240,19 @@ Window Azure
 <div class="area-inner">
 <div class="container">
 ## We are based in Bangkok
+{: .animated}
 ### We've been developing software for over ten years
+{: .animated }
+<div class="animated based-area-link">
 - [Test Driver Development](http://salween.com "Test Driver Development")    
 - [Continuous Integration](http://salween.com "Continuous Integration")    
 - [End to end testing ](http://salween.com "End to end testing")   
 - [Agile Methodologies](http://salween.com "Agile Methodologies")  
-- [Microsoft Gold Parners ](http://salween.com "Microsoft Gold Parners")    
+- [Microsoft Gold Parners ](http://salween.com "Microsoft Gold Parners")  
```

Comment:
Do the classes have to be on a `div`? If not, it should work to add `{: .animated .based-area-link }` after the final list item. That will add the classes to the `ul`.